extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.schema.json

```diff
@@ -17,6 +17,11 @@
     "decisioning-experienceevent-all": {
       "type": "object",
       "properties": {
+        "https://ns.adobe.com/experience/decisioning/propositionEventType": {
+          "title": "Associated Event",
```

Comment:
I'd like to think of some other word than "Associated" here - perhaps something that distinguishes this is a system  type whereas the other eventType is a cusotmer-defined type.
That said, I don't have a better idea ATM

Reply:
Would it be better to just call it "Proposition Event"?